### DISCUSSION TITLE
Lazer: New Acc Based SD Mod

### DISCUSSION THREAD

**Discussion by Glitch102x** [ppy/osu#17455] [Ideas] [answered]
So, we have SD, right? It helps with FCs, I think. Idk. Don't actually use the mod that much. It game ends you if you break combo so, I don't have much use for it. But what I *would* have a use for is an SD mod that acts based on accuracy. If you don't meet an accuracy threshold, you die. It helps with not submitting a score that you hate. A while ago, I was trying to SS a map with hidden that I already did with nomod. Problem is, I was uncertain if submitting a non-SS hidden score would override my SS. Might be a stupid concern, but it was there. This mod would prevent that. Maybe it calculates the maximum acc you can get from the map and when that drops below your set threshold, it game ends.

**Comment by bdach** [comment, marked as answer]
#13120
